source/security/index.html.md.erb

```diff
@@ -86,6 +88,8 @@ GOV.UK Pay is certified as fully compliant as a Level 1 Service Provider with
 PCI DSS version 3.2.1. All GOV.UK Pay partners must be compliant with PCI DSS,
 and must validate their compliance annually.
 
+Pay will be audited against PCI 4.0 in Summer 2024, at which point all relevant documentation will be updated. You may be asked to provide certain information from Pay as part of your own PCI compliance process, some of which may not be available until we have completed our PCI 4.0 transition work. In the interim this should not affect your ability to become PCI 4.0 compliant as there is a recognised transition period.
```

Comment:
```suggestion
GOV.UK Pay will be audited against PCI DSS v4.0 in summer 2024. After this audit, we'll update all relevant PCI DSS documentation. 

You may be asked to provide certain information from GOV.UK Pay as part of your own PCI DSS compliance process. Some of this information may not be available until we've completed our PCI DSS v4.0 transition work. This should not affect your ability to comply with PCI DSS v4.0 because there is a recognised transition period.
```

Changed this slightly to break up the paragraph and stay consistent with how we refer to PCI (as PCI DSS).

@Jonathan-Scott14 Who audits us for PCI? It might be good to mention in line 91 who that would be.